src/main/java/com/google/alpollo/SentimentServlet.java

```diff
@@ -18,12 +20,19 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String lyrics = request.getParameter("lyrics");
     AnalysisHelper helper = new AnalysisHelper();
-    Sentiment sentiment = helper.getSentiment(lyrics);
-    SongSentiment songSentiment = new SongSentiment(sentiment.getScore(), sentiment.getMagnitude());
-    
-    String json = gson.toJson(songSentiment);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
-  }
+    try {
+      Sentiment sentiment = helper.getSentiment(lyrics);
```

Comment:
I think it's better to move the try {...} inside the getSentiment() method, like I did on my PR. I don't consider it necessary for all the lines of code inside this doPost method.

Reply:
It's not possible because in catch block I send the error in response.